pypesto/logging.py

```diff
@@ -8,40 +8,53 @@
 import logging
 
 
-def log_to_console(level=None):
+def log_to_console(level: int = None, child: str = None):
```

Comment:
```suggestion
def log_to_console(level: int = None, id_: str = 'pypesto'):
```
makes it easier in case of "grandchildren" to just specify the full hierarchy "pypesto.{...}".

Reply:
Sure, went with `name` to match the `logging` module.